app/src/src/screens/menu/MenuHomeScreen.jsx

```diff
@@ -1,104 +1,140 @@
-import React, { useEffect, useState } from 'react';
+import React, { Component } from 'react';
 import {
   ActivityIndicator,
   Image,
   StyleSheet,
   Text,
   TouchableOpacity,
   View,
+  Platform,
 } from 'react-native';
 import Header from '../../components/views/Header';
-import { Avatar, Card, IconButton, Paragraph } from 'react-native-paper';
+import { Card, IconButton, Paragraph } from 'react-native-paper';
 import MenuLogout from '../../components/buttons/MenuLogout';
 import { routes } from '../../utils/routes';
 import { getUserData } from '../../utils/storageAccess';
+import ProfilePhotoView from '../../components/views/ProfilePhotoView';
 
 // A screen that allows the user to access menu options
-const MenuHomeScreen = ({ navigation }) => {
-  const [user, setUser] = useState(null);
+class MenuHomeScreen extends Component {
+  state = { user: {} };
 
-  useEffect(() => {
-    (async () => {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
If you're going to set the initial state outside the constructor, then there is no need to write out the constructor.